charts/home-assistant/values.yaml

```diff
@@ -41,4 +41,4 @@ config:
     allowPrivilegeEscalation: false
     capabilities:
       drop: [ "ALL" ]
-      add: []
+#      add: []
```

Comment:
Add should be explicitly empty no?

Reply:
https://github.com/FlorisFeddema/homelab/pull/1277 shows it is a problem

Reply:
Not empty array, empty prop

Reply:
https://github.com/FlorisFeddema/homelab/pull/1281, is now fixed 👍🏼